diff --git a/android/src/main/java/com/visioncameratextrecognition/VisionCameraTextRecognitionPlugin.kt b/android/src/main/java/com/visioncameratextrecognition/VisionCameraTextRecognitionPlugin.kt
index 1234567..abcdefg 100644
--- a/android/src/main/java/com/visioncameratextrecognition/VisionCameraTextRecognitionPlugin.kt
+++ b/android/src/main/java/com/visioncameratextrecognition/VisionCameraTextRecognitionPlugin.kt
@@ -45,7 +45,7 @@ class VisionCameraTextRecognitionPlugin(proxy: VisionCameraProxy, options: Map<S
         }
     }
 
-    override fun callback(frame: Frame, arguments: Map<String, Any>?): HashMap<String, Any>? {
+    override fun callback(frame: Frame, arguments: Map<String, Any>?): HashMap<String, Any?>? {
         val data = WritableNativeMap()
         val mediaImage: Image = frame.image
         val image =
